portfolio/src/main/webapp/background.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Carol's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class = "container">
+      <div class = "menu">
+        <ul>
+            <li class = "logo"> <a href="index.html"> <img src="images/logo.jpg"> </a> </li>
+            <li><a href="index.html"> About Me</a></li>
```

Comment:
Would be good to be consistent with spacing inside the `<a>` element:

- if there is not space before `</a>` then there should be no space after `<a>` either 
- if there a space after `<a>` then there should be a space before `</a>` too.